decompy/RL/ActionGenerator/ActionGenerator.py

```diff
@@ -0,0 +1,23 @@
+from decompy.RL.Action import Action
+
+
+class ActionGenerator:
+    """
+    Step 2: The Driver calls the ActionGenerator to get a list of possible actions for the current
+    llvm state it received in Step 1.
+    """
+
+    def __init__(self):
+        pass
+
+    def list_optimizations_actions(self, llvm_unop):
```

Comment:
The diagram for this was incomplete and incorrect. It should have had this as a private function along with list_swap_actions. It should expose only a "list_actions" function. Private functions are denoted with a "__" at the start